### ISSUE TITLE
AI Governance and Security

### ISSUE THREAD

**Issue by yoonghan** [yoonghan/yoonghan.github.io#517]
Generative AI Security Scope

Scope 1 - Consumer App (i.e. ChatGPT), no ownership
Scope 2 - Enterprise App, Using an app/Saas with GenAI, few ownership
Scope 3 - Pre-trained Model, Build your app on a versioned model (i.e. Amazon Bedrock)
Scope 4 - Fine tune Model, base on existing Foundation Model. Partial ownership
Scope 5 - Self-trained Model, Do own Foundation model, all ownership

Each has own security level.

**Comment by yoonghan** [comment]
AI practices that AWS has implemented for their AI services?
* AI Service Cards
It can be retrieved with SageMaker

**Comment by yoonghan** [comment]
interpretability for a model = Model that provides transparency so human can explain model output.

**Comment by yoonghan** [comment]
"Design for amplified decision-making" is correct because it emphasizes creating tools and environments that enhance decision-making capabilities, especially in high-pressure situations, helping prevent mistakes. This principle aligns with the objective of Human-Centered Design by focusing on user needs in stressful environments, ensuring better outcomes.

**Comment by yoonghan** [comment]
## Monitoring AI System

* Performance metrics 
   * Model accuracy - ratio of positive predictions
   * Precision - ratio of true prediction (both true/false)
   * Recall - positive prediction vs actual prediction
   * FI-score - average score
   * Latency
* Infrastructure Monitoring - CPU/GPU/Failure
* Bias and fairness (can detect via SageMaker clarify)
* Robustness covers as fairness too as, if system failed to cover ethnics

**Comment by yoonghan** [comment]
Security - AWS does not manage AI data encryption. Customer job.

**Comment by yoonghan** [comment]
## Assessing Data Quality
- Completeness
- Accuracy
- Timeliness
- Consistency
- Data profiling
- Data lineage

**Comment by yoonghan** [comment]
AI Poisoning:
Mechanism:
Poisoning refers to the intentional introduction of malicious or biased data into the training dataset of a model which leads to the model producing biased, offensive, or harmful outputs (intentionally or unintentionally).

Prompt Leaking: 
Mechanism:
Prompt Leaking refers to the unintentional disclosure or leakage of the prompts or inputs used within a model. It can expose protected data or other data used by the model, such as how the model works.

**Comment by yoonghan** [comment]
**Data residency** - data location is stored.
**Data retention** - duration data is stored.

**Comment by yoonghan** [comment]
https://aws.amazon.com/blogs/security/securing-generative-ai-an-introduction-to-the-generative-ai-security-scoping-matrix/
